### PR TITLE
143 updates based on feedback

### PR DESCRIPTION
**Pull request by chadyj**
Added `Security and Privacy Implications` section and updated idea and iterations.

### CODE REVIEW

File: ideas/143-seamless-login.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### Security and Privacy Implications
+
+Potentially, both Touch ID ("fingerprint authentication") and Face ID can improve security by allowing users to have stronger passwords (the current experience encourages to have a weak password). Of course, it has some disadvantages, but we're going to require passwords anyway for important operations, so the goal is to improve security anyway.
+Moreover, options like "Require authentication every time I open the application" will improve both security and privacy for sure – right now we ask password only in the case user was accidentally logged out.  
+
+Of course, there are some open questions:
+1) We should discuss options like requiring passwords to log in once in a while even if biometrical authentication is used;
+2) Screens like "Settings" or "Backup your Seed Phrase" should be protected by password anyway;
+3) We need to discuss the "level of freedom" – what should be the options? 
+4) How can we encrypt/decrypt Realm in the situation when we don't have a password?
```
Comment (cammellos) [inline ideas/143-seamless-login.md]: What does it mean exactly? 

As far I understand we don't need a password to encrypt realm, just a key which is in the keychain and can be randomly generate ( which I take can be unlocked using touch id/face id).

File: ideas/143-seamless-login.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Status frequently logs out users when the app is in the background and forces users to log in again when opening the app. Being repeatedly and unexpectedly logged out has led to frustration and complaints (see Instabug and internal reports) which negatively impact the user experience and potentially contributes to churn. Also, users are encouraged to have complex passwords but this can be cumbersome to enter.
+
+By reducing friction around the critical point of returning to the app users can spend more time in Status which will help DAU and retention (Q2 OKR).
+
+This idea seeks to make logging in seamless through both technical and UX methods.
+
+
+### Product Description
+
+The Status login process should smooth as possible, and this can be achieved with the (optional) use of device security features such as Touch ID or a pin code. In addition to the password prompt, a user can use choose to securely use their fingerprint. This will be set up during onboarding and toggled in settings.
```
Comment (adambabik) [inline ideas/143-seamless-login.md]: _a user can ~~use~~ choose to_

File: ideas/143-seamless-login.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+For future iterations, this swarm will research the security implications and explore enhancements around logging in, logging out, and recovering from crashes.
+
+### Security and Privacy Implications
+
+Potentially, both Touch ID ("fingerprint authentication") and Face ID can improve security by allowing users to have stronger passwords (the current experience encourages to have a weak password). Of course, it has some disadvantages, but we're going to require passwords anyway for important operations, so the goal is to improve security anyway.
+Moreover, options like "Require authentication every time I open the application" will improve both security and privacy for sure – right now we ask password only in the case user was accidentally logged out.  
+
+Of course, there are some open questions:
+1) We should discuss options like requiring passwords to log in once in a while even if biometrical authentication is used;
+2) Screens like "Settings" or "Backup your Seed Phrase" should be protected by password anyway;
```
Comment (adambabik) [inline ideas/143-seamless-login.md]: 👍 

`s/anyway/always`?

File: ideas/143-seamless-login.md (hunk 1 above)
Comment (alwx) [reply ideas/143-seamless-login.md]: You're right, let's remove this one.

**Pull request merged**